wikimedia/wikipedia-ios
# Scale down Picture of the Day sample entry image to match the size of requested images

**Fixes Phabricator ticket:** 
https://phabricator.wikimedia.org/T263271

### Notes
While testing #3724, I noticed our widgets would flicker in the widget gallery even if the POTD widget had the example image set. The example image in the project is much larger than what we request for POTD and was causing the process to be terminated. Even though the size of the JPEG is 1.6MB it takes up much more memory when being decompressed for display due to the dimensions being 5309 by 3543.

### Test Steps
Getting a debugger consistently attached to the widgets is still an inconsistent process so you might have better luck with different steps.

1. Erase all content and settings on a simulator (I used the iPhone 11 Pro simulator)
2. Clean the build folder
3. Build and run the WidgetsExtension with `_XCWidgetKind` set to `org.wikimedia.wikipedia.widgets.potd` and `_XCWidgetFamily` set to `large`
4. After the widget launches and the debugger is attached, enter home screen edit mode and open the Wikipedia widget gallery
5. Observe memory usage graph
6. Realize it's using _more_ memory after these changes and close the PR

It's possible this test can be automated with `XCTMemoryMetric`
